src/providers/ibmcloud/mod.rs

```diff
@@ -93,6 +95,21 @@ impl IBMGen2Provider {
         Ok(output)
     }
 
+    /// Return the path to the vendordata directory.
+    fn vendordata_dir(&self) -> PathBuf {
+        self.drive_path.clone()
+    }
+
+    /// Read vendordata file into a string
+    fn read_vendordata(&self) -> Result<String> {
+        let filename = self.vendordata_dir().join("vendor-data");
+        let mut file =
+            File::open(&filename).chain_err(|| format!("Failed to open vendordata '{:?}'", filename))?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents).expect("Unable to read vendordata");
```

Comment:
Please do not expect/unwrap in main logic (i.e. everywhere outside of tests), prefer bubbling up the error instead.